conf/test.config

```diff
@@ -44,7 +44,7 @@ params {
     malt_save_reads                       = true
     kraken2_save_reads                    = true
     centrifuge_save_reads                 = true
-    diamond_save_reads                    = true
+    run_profile_standardisation           = true
```

Comment:
Do we want to standardise only, merge only, or offer both taxpasta commands?

Reply:
I think we said that we would offer merge only, right @jfy133 ?

Reply:
Merge only, as it does essentially the same thing. I don't think it's necessary to spit out single-sample files as they will already have this (with 'richer' info) from the default outputs.

Reply:
note `run_profile_standardisation` is the referring to the subworkflow, rather than specifically taxpasta